common/src/commonMain/kotlin/com/cuhacking/atlas/common/Example.kt

```diff
@@ -1,3 +1,4 @@
 package com.cuhacking.atlas.common
 
-expect fun getMessage(): String
```

Comment:
The other targets (ios and js) still need the `expect` function. We can keep this, and the deleted `Example.kt` from the android sourceset for now. Otherwise, this breaks some of the builds.